.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "6"
```

Comment:
For htcp-purge we used Node 8, 10, and 12... should we add that here? Also, do we want to move this away from Travis to Github Actions?

Reply:
hm... regarding 8 - we don't really use or plan on using 8, so.. meh.. regarding gh actions.. why not.